Arduino/CN470RP2/TransmissionTestABPLoRa/TransmissionTestABPLoRa.ino

```diff
@@ -0,0 +1,113 @@
+
+#include <LoRaWan.h>
+// THOSE ARDUINO SAMPLE REQUIRES CHANGES IN THE ARDUINO CODE TO RUN DESCRIBED IN THE MAIN REPO
+// THE CONCENTRATOR IS EXPECTED TO HAVE THE FOLLOWING FREQUENCIES ACTIVATED
+// 498.3, 498.7, 498.9, 499.1, 499.3, 499.5, 499.7, 499.9
+// please refer to the repo documentation for further information
+
+//set to true to send confirmed data up messages
+bool confirmed = true;
+//application information, should be similar to what was provisiionned in the device twins
+char * deviceId = "46AAC86800430028";
+char * devAddr = "0228B1B1";
+char * appSKey = "2B7E151628AED2A6ABF7158809CF4F3C";
+char * nwkSKey = "3B7E151628AED2A6ABF7158809CF4F3C";
+
+
+/*
+  iot hub ABP tags for deviceid: 46AAC86800430028
+    "desired": {
+    "AppSKey": "2B7E151628AED2A6ABF7158809CF4F3C",
+    "NwkSKey": "3B7E151628AED2A6ABF7158809CF4F3C",
+    "DevAddr": "0228B1B1",
+    "GatewayID" :"",
+    "SensorDecoder" :"DecoderValueSensor"
+    },
+*/
+
+//set initial datarate and physical information for the device
+_data_rate_t dr = DR6;
+_physical_type_t physicalType = CN470PREQUEL ;
+
+//internal variables
+char data[10];
+char buffer[256];
+int i = 0;
+int lastCall = 0;
+
+
+void setup(void)
+{
+  SerialUSB.begin(115200);
+  while (!SerialUSB);
```

Comment:
That's a very tight loop, I assume we do this everywhere? Any way we could sleep for a bit before checking again?

Reply:
What is your worry there? this code would trigger onlywhen we actively open the serial

Reply:
"we actively open the serial"? we do this here right? We wait for it to become available, no? If it is not, this is a very tight loop. I think we have this everywhere and it seems to work, so I assume it's ok?

Reply:
Yeah, this is a copy from their sample, to be honest I don't know how much is the backoff/retry on this.

Reply:
If you see a problem with it I can make some research but this thing has always worked so far and it's in the sample repo